portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,86 @@
+<!-- 
+MAIN HTML
+command week 1: teachme ~/software-product-sprint/walkthroughs/week-1-web-development/portfolio-walkthrough.md
+command week 2: teachme ~/software-product-sprint/walkthroughs/week-2-server/project-1-walkthrough.md
+portfolio : cd \ ~/software-product-sprint/portfolio
+random message: teachme \
+    ~/software-product-sprint/walkthroughs/week-2-server/project-1-walkthrough.md
+
+https://nnietovega-sps-spring21.uc.r.appspot.com
+execute: mvn package exec:java
+href="https://github.com/naominietov"
+href="https://www.linkedin.com/in/naominietov"
+-->
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title> Naomi's Portfolio </title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script src="script.js"> </script>
   </head>
-  <body>
+
+  <!--Background color --><body bgcolor="#ffd9da">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+
+    <h1 style="background-color:#89023e; font-family:helvetica; color:white; text-align:center;"> 
+        Naomi's Portfolio &#128105;&#127995;&#8205;&#128187;
+        </h1>
```

Comment:
From a code styling/maintenance perspective, when an HTML element (e.g. this h1) spans multiple lines, I think it's nice to line up the closing indicator with the opening one. It's OK to ignore this on the review if you don't wish to update it, but I just wanted to give you the feedback as a learning opportunity.